qucs/CMakeLists.txt

```diff
@@ -266,7 +266,7 @@ ADD_EXECUTABLE( ${QUCS_NAME} MACOSX_BUNDLE WIN32
 # Tell CMake which libraries we need to link our executable against.
 #
 TARGET_LINK_LIBRARIES( ${QUCS_NAME}  components diagrams dialogs paintings extsimkernels spicecomponents qt3_compat ${QT_LIBRARIES} )
-SET_TARGET_PROPERTIES(${QUCS_NAME} PROPERTIES POSITION_INDEPENDENT_CODE TRUE)
```

Comment:
The duplicate option here is done intentionally. The Qucs application may be built separately from the utilities for debug purposes. If toplevel Cmake is not accessible the build will fail without setting PIC option here.

Reply:
Yes, the problem with PIC flag has been resolved too.